tests/type-tests/test/orchestration.test-d.ts

```diff
@@ -3,24 +3,26 @@ import {
   OrchestrationClient,
   buildAzureContentSafetyFilter,
   buildDocumentGroundingConfig,
-  buildLlamaGuardFilter,
-  buildDpiMaskingProvider
+  buildLlamaGuard38BFilter,
+  buildDpiMaskingProvider,
+  buildTranslationConfig
 } from '@sap-ai-sdk/orchestration';
+import type { TranslationReturnType } from '@sap-ai-sdk/orchestration/src/orchestration-types.js';
```

Comment:
[req] This is part of public api no? Can be imported from `@sap-ai-sdk/orchestration`

Reply:
Oh this is auto imported wrong. Good catch.